### ISSUE TITLE
Dead Project?

### ISSUE THREAD

**Issue by Sevyn13** [jakibaki/In-Home-Switching#51]
Is this project dead?

**Comment by mirh** [comment]
[Yes.](https://gbatemp.net/threads/in-home-switching-stream-your-favorite-pc-games-to-your-nintendo-switch.526108/page-23#post-8625403)

**Comment by d0x360** [comment]
Why...

**Comment by DevL0rd** [comment]
@jakibaki I have rewrittent the streaming app from scratch in node.js in preperation to make a better UI with electron. It's fully functional now if you want to try it.

I'm going to do the same with the Switch app. I've already added touch controls and it works nicely.
Next I am going to rewrite the UI. But I want to get your permission to take over the project and start working on it on my repository here. [SkyNX](https://github.com/DevL0rd/SkyNX)

This repository has all the working code for this re-write. But I am avoiding compiling it until I have your permission. Let me know ey? I'd love to revive this.

**Comment by jakibaki** [comment]
Oh nice, awesome work!  
You definitely have permission to use it.

However we've been planning to potentially renew the app at some point so I am in turn gonna ask you for permission to merge some of your changes back into the original project (obviously with full credit).

**Comment by DevL0rd** [comment]
Of course! The streamer isn't finished yet. As it doesn't have a UI. But i'm seperating the streamer into a service. And the using the UI to configure it, so you can close it and it still runs. Or so it can start at startup waiting for the app to be launched. So it can always connect at anytime.

I have auto reconnect working and in my dev build atm the sleep issue is fixed.
But my build of the app, not commited, already has a new UI. It's completely different. Same with the streamer app. With the streamer as a library / service you should be able to write any UI for it.

But the app will be difficult to push changes back if it gets radically different, and your not updating your streamer to read the new packets. (Planning on sending rotation data and sending vibrations to the console)

Perhaps this version can be marked as outdated or abandoned? And I can give you full permissions to this new repo, because I don't want two competing versions of the same thing. I'd definitely like you to keep working on it at some point to help renew it, but I'm already doing it now and would like the ability to commit my changes whenever and approve changes.

Anyway we should certainly figure this out because, I'm renewing it now lol and it's only taken me a total of maybe 18 hours of work to get this far haha I'd like to continue it definitely.

**Comment by DevL0rd** [comment]
Ok here is the first release.

Let me know if there are issues :p

[https://github.com/DevL0rd/SkyNX/releases/tag/0.0.1](https://github.com/DevL0rd/SkyNX/releases/tag/0.0.1)

**Comment by DevL0rd** [comment]
> Oh nice, awesome work!
> You definitely have permission to use it.
> 
> However we've been planning to potentially renew the app at some point so I am in turn gonna ask you for permission to merge some of your changes back into the original project (obviously with full credit).

Oh wait I see what you mean. Yeah take any code you need XD I'm writting graphics librarys right now for the UI and wrote a little physics loop for some bg animations. I'll put up a new screenshot of the UI soon.

**Comment by d0x360** [comment]
I love you guys, ty so much

**Comment by DevL0rd** [comment]
@d0x360 NP!

@jakibaki I have been taking all your users that have reported issues.
If they have reported issues here, hopeufully they will help on the new repo. Whenever you want the project back let me know! I have also given you full permissions to the new repo.

**Comment by d0x360** [comment]
Awesome, thanks and I'll be letting people know about it as well

**Comment by DevL0rd** [comment]
## SkyNX App
![SkyNX App](https://github.com/DevL0rd/SkyNX/raw/master/Screenshots/App.jpg "SkyNX App")

## SkyNX Streamer
![SkyNX Streamer](https://github.com/DevL0rd/SkyNX/raw/master/Screenshots/Streamer.png "SkyNX Streamer")

**Comment by DevL0rd** [comment]
Had fun with the buble physics XD

**Comment by DevL0rd** [comment]
@jakibaki Daw thanks for the notice on the read me.
Got the audio working awesome and fixed high cpu usage in the checkFrameAvail loop.

The audio buffer tho is't always perfectly filled I think. But IDK the extent of the audio code.
What where you looking into when you where trying to find the issues with audio?

Also let me know if you need something changed on my readme like wordings or anything.
